Consuming.md

```diff
@@ -0,0 +1,28 @@
+# Consumers 
+
+The information below is meant for users of the OpenTDF platform.
+
+To contribute/develop, see [here](./Contributing.md).
+
+## Running the Platform Locally
+
+1. Initialize KAS Keys ```.github/scripts/init-temp-keys.sh -o kas-keys```
```

Comment:
We need to make this accessible to people using the project via `docker run` or `go run`. This would could mean making the script available directly, or including it as an application subcommand.

Reply:
I think that's a great idea, absolutely 👍 That sort of change could also help address some parts of https://github.com/opentdf/platform/issues/758

Reply:
I think this is resolved via https://github.com/opentdf/otdfctl/issues/127 👍